### ISSUE TITLE
No such file or directory when publishing a site

### ISSUE THREAD

**Issue by joerter** [jacobobryant/platypub#88]
I'm trying out Platypub and trying to publish my site. When I click on publish, platypub encounters an error in `generate!` when trying to rsync the themes directory. The exact error is saying that the directory created in `storage/deploys/{random-uuid}` doesn't exist. I can see on my file system that the directory doesn't exist, but the call to `io/file` in `publish` should create the directory, right?

I'm sure I'm just doing something wrong here, but if there is a code fix, I would love to take a crack at it if you can point me in the right direction. Thanks!

**Comment by jacobobryant** [comment]
Hm, looks like rsync won't create nested folders for the destination like I thought it did. It probably works for me because I happened to create the `storage/deploys/` folder in some other way while developing platypub; if I deleted it I'd probably get the same error. How about copy [this line](https://github.com/jacobobryant/platypub/blob/8dd01fa4c07698d85eed087e52c949748fe31a64/src/com/platypub/feat/sites.clj#L85) so it runs right before the rsync and see if that solves the problem? Feel free to submit a PR if so.
